### PR TITLE
init a single node docker swarm

### PR DESCRIPTION
**Pull request by reaperhulk**
As is our custom with ansible scripts this is tested by me running it and confirming it works properly.

### CODE REVIEW

File: ansible/playbook.yml [hunk 1]
```diff
@@ -39,6 +39,11 @@
           state: started
           enabled: yes
       with_items: "{{ services }}"
+    - command: docker info
+      register: docker_info
+    - name: "set up a single node docker swarm"
+      command: docker swarm init --advertise-addr bond0
```
Comment (alex) [inline ansible/playbook.yml]: Can you please add a comment explaining just what the fresh hell `--advertise-addr bond0` means?

File: ansible/playbook.yml [hunk 2]
```diff
@@ -39,6 +39,16 @@
...
       with_items: "{{ services }}"
+    - command: docker info
+      register: docker_info
+    - name: "set up a single node docker swarm"
+      # docker swarm needs a single address to advertise its existence.
+      # Our server, however, has two addresses. By specifying an explicit
+      # named interface on the server (bond0 in this case) we make docker
+      # happy. If we stop using OnMetal in the future this name will
+      # change.
+      command: docker swarm init --advertise-addr bond0
```
Comment (alex) [inline ansible/playbook.yml]: Ok, now that I understand what this does, would you mind moving `bond0` to be in a `var`?

Comment (reaperhulk) [inline ansible/playbook.yml]: Sure, lemme figure out what a var is in ansible :smile:

Comment (alex) [inline ansible/playbook.yml]: grep for it in the file

**Pull request merged**